src/lib/openapi/spec/application-schema.ts

```diff
@@ -7,31 +7,51 @@ export const applicationSchema = {
     required: ['appName'],
```

Comment:
Are there other properties that are always present?

Reply:
No, they're all optional in the type we use to resolve it as well, except for the appName.